File: packages/python/scripts/eval_corpus.py
#!/usr/bin/env python3
from __future__ import annotations

import json
import sys

from chronow import evaluate_corpus_file


if len(sys.argv) != 2:
    print("usage: python scripts/eval_corpus.py <cases-file>", file=sys.stderr)
    raise SystemExit(2)

result = evaluate_corpus_file(sys.argv[1])
print(json.dumps(result, indent=2, sort_keys=True))

File: packages/ts/scripts/eval-corpus.mjs
import { evaluateCorpusFile } from "../dist/index.js";

const casesFile = process.argv[2];
if (!casesFile) {
  console.error("usage: node scripts/eval-corpus.mjs <cases-file>");
  process.exit(2);
}

const result = evaluateCorpusFile(casesFile);
process.stdout.write(`${JSON.stringify(result, null, 2)}\n`);
